RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -2276,6 +2276,47 @@ boolean isUnclePAvailable()
 	return !page_text.contains_text("You don't know where a desert beach is");
 }
 
+boolean isDesertAvailable()
+{
+	//Is this workaround still needed or is mafia correctly recognizing desert is available in koe?
+	if(in_koe())
+	{
+		auto_log_info("The desert exploded so no need to build a meatcar...");
+		set_property("lastDesertUnlock", my_ascensions());
+	}
+	
+	return get_property("lastDesertUnlock").to_int() == my_ascensions();
+}
+
+boolean inKnollSign()
+{
+	return $strings[Mongoose, Vole, Wallaby] contains my_sign();
+}
+
+boolean inCanadiaSign()
+{
+	return $strings[Marmot, Opossum, Platypus] contains my_sign();
+}
+
+boolean inGnomeSign()
+{
+	return $strings[Blender, Packrat, Wombat] contains my_sign();
+}
```

Comment:
unnecessary. Should use `knoll_available()`, `canadia_available()` & `gnomads_available()` instead.

Reply:
gnomads_available() will return misleading results unless you have access to the desert.
and all 3 zones are not accessible in kingdom of exploathing.